source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts

```diff
@@ -69,21 +59,28 @@ export class BerlinContainer extends HostContainer {
         return this._colorTheme === ColorTheme.Light ? "#D2D2D2" : "#616161";
     }
 
-	get targetVersion(): Adaptive.Version {
-		return Adaptive.Versions.v1_6;
-	}
-
-    public static allContainers(): BerlinContainer[] {
-        let containers: BerlinContainer[] = [];
-        for (const size in ContainerSize) {
-            for (const theme in ColorTheme) {
-                containers.push(new BerlinContainer(size as ContainerSize, theme as ColorTheme));
-            }
-        }
-        return containers;
+    get targetVersion(): Adaptive.Version {
+        return Adaptive.Versions.v1_6;
     }
 
     get isFixedHeight(): boolean {
         return true;
     }
+
+    set containerSize(value: ContainerSize) {
```

Comment:
I'm wondering if we made the host data editor read-only.
In reality, user can't make any change on it.

![image](https://user-images.githubusercontent.com/6290692/185210722-9f0998b3-69b6-4ddc-b4c4-c51e48a2edcb.png)

Reply:
@licanhua Is there any scenario where the host would want to use the designer?

Reply:
I don't see any except that the one who owns the host and want to prototype the host data feature.

It's reasonable for me to remove `host data editor` or minimize it by default since the toolbar already includes duplicated information.

![image](https://user-images.githubusercontent.com/6290692/185223540-9864fc7e-43cf-4946-b8ab-cc509b620066.png)

Reply:
That's a good call out. I like the idea of minimizing it especially since 3 data editors looks a little cluttered.

I also think its fair to note that adaptivecards developers could end up using `$host` parameters for other functionality as well. Since the designer and the templating library aren't bound to the widgets platform, I'm not sure if we should limit the functionality of the JSON editor to only use size and theme

Reply:
I agree with Anna here -- `$host` parameters aren't required to be coming from the Widget Dashboard, but is rather a generic data-passing mechanism

Reply:
Resolving since the $host parameter data editor is needed for non-Widget users.